app/views/appointments/index.csv.erb

```diff
@@ -0,0 +1,30 @@
+<%= [
+  "Patient name",
+  "Gender",
+  "Age",
+  "Days overdue",
+  "Last BP",
+  "Last BP taken at",
+  "Last BP date",
+  "Risk level",
+  "Patient address",
+  "Patient village or colony",
+  "Patient phone"
+].to_csv -%>
+<%- @appointments.group_by { |a| a.patient.latest_blood_pressure.facility }.each do |facility, facility_appointments| %>
```

Comment:
There are a couple of `<%-` and `-%>` ligatures.
Should these `-`s be `=`s instead?

Reply:
These suppress printing of newlines and whitespace in ERB:
https://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails